test/unit/src/core/Raycaster.tests.js

```diff
@@ -141,7 +145,7 @@ export default QUnit.module( 'Core', () => {
 
 			const raycaster = new Raycaster();
 			const rayDirection = raycaster.ray.direction;
-			const camera = new PerspectiveCamera( 90, 1, 1, 1000 );
+			const camera = new PerspectiveCamera( 90, 1, 0.1, 1000 );
```

Comment:
Previously the ray direction was being calculated with the "origin" being calculated as "0, 0, 0" which is used to calculate a delta vector in the `setFromCamera` function. With the change in this PR the origin is at the near plane which is 1 unit from the camera origin resulting in a higher epsilon which Number.EPSILON was not high enough to test for. Lowering the near value lowers the values used in the origin vector and therefore the error in the calculations. From the [MDN page on Number.EPSILON](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/EPSILON):

> The Number.EPSILON static data property represents the difference between 1 and the smallest floating point number greater than 1.